suite-native/assets/src/assetsSelectors.ts

```diff
@@ -0,0 +1,74 @@
+import { createSelector } from '@reduxjs/toolkit';
+import BigNumber from 'bignumber.js';
+
+import { Network, networksCompatibility, NetworkSymbol } from '@suite-common/wallet-config';
+import { selectAccounts, selectCoins } from '@suite-common/wallet-core';
+import { toFiatCurrency } from '@suite-common/wallet-utils';
+import { selectFiatCurrency } from '@suite-native/module-settings';
```

Comment:
Can you please accept fiat currency as input arg instead? This dependency is not good and input arg is easiest solution here.